aiohttp/client.py

```diff
@@ -654,6 +683,60 @@ def loop(self):
         """Session's loop."""
         return self._loop
 
+    @property
+    def on_request_start(self):
+        return self._on_request_start
+
+    @property
+    def on_request_redirect(self):
+        return self._on_request_redirect
+
+    @property
+    def on_request_end(self):
+        return self._on_request_end
+
+    @property
+    def on_request_exception(self):
+        return self._on_request_exception
+
+    # connector signals
+
+    @property
+    def on_request_queued_start(self):
+        return self._connector.on_queued_start
```

Comment:
I think every session should have own signals.
Connector might be shared between sessions by `connector_owner=False` or `session.detach()`.
Shared subscriptions make a mess: nobody know when recipient is dead.

Reply:
Oks I can see right now the idea of the `connector_owner` parameter, my fault I havent check it and it head me to a wrong implementation. If at last the connector is shared between sessions its absolutely necessary don't mess the signals.

In another way worries me a bit the `connector_owner` implementation, leaving to the user's hand the power of making it `True` or `False`, meaning that in case the user gives an alternative connector but forget to pass the `connector_owner` set as `False`, when the session is gonna be closed this automatically will close the connections of the connector that might be shared by another `Session`. 

Should the `connector_owner` param handled internally by the `Session` object` ?